### ISSUE TITLE
Cost model details page crashes after refresh

### ISSUE THREAD

**Issue by dchorvat1** [project-koku/koku-ui#957]
**Describe the bug**
After refreshing cost model details, the page crashes

**To Reproduce**
1. Navigate to cost model details
2. Refresh page
3. See error

**Expected behavior**
Page should display correctly

**Screenshots**
*Browser console output:*
![Screenshot from 2019-08-23 12-32-27](https://user-images.githubusercontent.com/46324552/63586826-a3809e80-c5a2-11e9-8cef-7e7e30260b92.png)
*The page:*
![Screenshot from 2019-08-23 12-32-36](https://user-images.githubusercontent.com/46324552/63586846-b72c0500-c5a2-11e9-96ad-4517080938f5.png)

**Desktop (please complete the following information):**
 - OS: rhel
 - Browser chrome
 - Version 75

**Additional context**
Tested and reproduced on both CI and QA

**Comment by boaz0** [comment]
That's odd. I can't reproduce it on my environment. In addition, looking at the error message I am not sure what is the problem. I do see a "TypeError" that is related to fetching that information but I don't know where exactly it happens.

Can you provide more network logs (request that were sent and etc)?

Thanks a lot @dchorvat1

**Comment by dchorvat1** [comment]
@boaz0 From the network logs on my side, all the api requests were successful. I assume you were trying to reproduce it on your local environment? If that's the case, could you try it against live CI/QA envs? That's where I encountered it

**Comment by dchorvat1** [comment]
At first I encountered this using my account, then I reproduced it with Jen's account, with same results

**Comment by boaz0** [comment]
@dlabrecq @dchorvat1 

I opened a ticket on insights JIRA. They did reproduce it successfully and I'm waiting for other instructions or ideas how to debug it.

I thought it might be related to how we lazy loading the pages but I think it's related to the platform.

I will keep you updated.

Thanks.

**Comment by boaz0** [comment]
Some updates:
 - I confirmed that this problem is related to Webpack.
 - Patternfly table was not loaded when going to cost models page
 - I have no idea why it's not loaded through the cost models page but there were open tickets in Webpack complaining about scoped packages aren't served by Webpack sometimes.
 - Changing the webpack config js so I explicitly add patternfly to the vendor.js fixed it
 - It looks like insights has a package to setup webpack config js which we should use - although I'm not sure that it fixes the problem but I would like to check it.

In conclusion, I'm going to create a pull request with that fix but we should check insights https://github.com/RedHatInsights/frontend-components/tree/master/packages/config to config bundles in our projects

@dlabrecq @dchorvat1
